### ISSUE TITLE
Test failure: JIT\\Regression\\VS-ia64-JIT\\V1.2-M02\\b108129\\b108129\\b108129.cmd

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#36968]
failed in job: [runtime-coreclr jitstress2-jitstressregs 20200524.1](https://dev.azure.com/dnceng/public/_build/results?buildId=658473&view=ms.vss-test-web.build-test-results-tab&runId=20435328&resultId=107217&paneView=debug)

Error message
~~~
Return code: 1
Raw output file: C:\h\w\A6980933\w\A2CF096D\e\JIT\Regression\Reports\JIT.Regression\VS-ia64-JIT\V1.2-M02\b108129\b108129\b108129.output.txt
Raw output:
BEGIN EXECUTION
 "C:\h\w\A6980933\p\corerun.exe" b108129.dll 
Int32 Const = 7
Int64 Const = -4294967289
Float Const = 7.777777
Double Const = 7.777777
FAILED: GetInt64Const()!=7
GetInt64Const() is 222885231881879559
Expected: 100
Actual: 1
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=C:\h\w\A6980933\p
> C:\h\w\A6980933\w\A2CF096D\e\JIT\Regression\VS-ia64-JIT\V1.2-M02\b108129\b108129\b108129.cmd
Expected: True
Actual: False


Stack trace
   at JIT_Regression._VS_ia64_JIT_V1_2_M02_b108129_b108129_b108129_._VS_ia64_JIT_V1_2_M02_b108129_b108129_b108129_cmd() in F:\workspace\_work\1\s\artifacts\tests\coreclr\Windows_NT.x86.Checked\TestWrappers\JIT.Regression\JIT.Regression.XUnitWrapper.cs:line 137104
~~~

**Comment by BruceForstall** [comment]
Failure mode: `CoreCLR Windows_NT x86 Checked jitstress2_jitstressregs4 @ Windows.10.Amd64.Open`

@dotnet/jit-contrib

**Comment by tannergooding** [comment]
@CarolEidt, was this fixed by https://github.com/dotnet/runtime/pull/36994 ?

Given there were 287 failures fixed by https://github.com/dotnet/runtime/pull/36994, I imagine there are a few of these open issues that could be closed now, the assert was just slightly different for each and so they all got shared.

**Comment by CarolEidt** [comment]
> was this fixed by #36994 ?

Unknown. This was a failure in a different stress job than the ones I fixed with #36994. I'm now working on the remaining failures (2 tests x 2 x64/ux platforms with the same failure mode) in the `runtime-coreclr jitstressregs` leg, and then I'll rerun the `runtime-coreclr jitstress2-jitstressregs` leg to see what remains.

**Comment by CarolEidt** [comment]
Fixed by #36994 or #37062
